dev-libs/blazesym_c/files/0.1.1-add-capi-feature-to-enable-building-with-cargo-c.patch

```diff
@@ -0,0 +1,49 @@
+From 469cd07ff8e83cbed05ca3bbfce9a887efa7552f Mon Sep 17 00:00:00 2001
```

Comment:
1. Add direct link to the commit URl
2. run `scrub-patch` over the patch to make it a little bit slimmer.

Reply:
done. didn't know about scrub-patch!